packages/roosterjs-editor-dom/lib/table/pasteTable.ts

```diff
@@ -0,0 +1,59 @@
+import moveChildNodes from '../utils/moveChildNodes';
+import VTable from './VTable';
+import { NodePosition, TableOperation } from 'roosterjs-editor-types';
+
+/**
+ *
+ * Pastes a table inside another, modifying the original to create a merged one
+ * @param currentTd The cell where the cursor is in the table to paste into
+ * @param rootNodeToInsert A Node containing the table to be inserted
+ * @param position The position to paste the table
+ * @param range The selected range of the table
+ *
+ * Position and range are here for when table selection allows to move pivot point
+ */
+export default function pasteTable(
+    currentTd: HTMLElement,
```

Comment:
Change the parameter type to be HTMLTableCellElement so you can avoid type casting in line 24